common/protob/messages-cardano.proto

```diff
@@ -221,9 +218,11 @@ message CardanoPoolParametersType {
     required uint64 margin_numerator = 5;                 // pool margin numerator
     required uint64 margin_denominator = 6;               // pool margin denominator
     required string reward_account = 7;                   // bech32 reward address where the pool receives rewards
-    repeated CardanoPoolOwnerType owners = 8;             // pool owners list
-    repeated CardanoPoolRelayParametersType relays = 9;   // pool relays list
+    // repeated CardanoPoolOwnerType owners = 8;             pool owners list
```

Comment:
It was intentional, because this "message" is used both in the old and new firmware. But maybe I should have left the old list fields in place, just not use them? Or should I duplicate the whole message and have separate old and new versions? I'm still a bit confused by the whole backwards compatibility topic. :confused:

Reply:
Oh, I see. I thought this was a new message (I even double checked and came to that conclusion 😐). In that case, IMHO the old fields should be removed (commented out) and the new fields should have ids 11 and 12.

Reply:
:eyes: Oops the id's are messed up indeed.
But if I just remove the old fields, then Trezor Connect won't be able to construct an old `CardanoSignTx` with all the data included. Or will it?
I'm leaning towards leaving the old fields with a [deprecated](https://developers.google.com/protocol-buffers/docs/proto3#options) flag. We can leave them empty, as repeated fields seem to be optional by default. And add the new required counts, as they can be easily filled even when constructing an old message, and shouldn't bother anything there.

Reply:
You are right, Trezor Connect wouldn't be able to construct the old message. Let's keep the `deprecated` flag there with the new fields added. Have you tried looking into Trezor Connect to see if the flag has some effect? If not, don't worry about it, I'll check later.